app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -812,6 +812,11 @@ def rbac_field_changed(rec_type)
         page.replace(@refresh_div, :partial => @refresh_partial) if @refresh_div
         bad = false
       else
+        # only do following if groups of a user change (adding/editing a user)
+        if x_node.split("-").first == "u" || x_node == "xx-u"
+          page.replace("group_selected",
+                       :partial => "rbac_group_selected")
```

Comment:
can you add `ops/` just to be sure? :) (`rbac_group_selected` will find that one first, but wouldn't stop there if it didn't so.. just futureproofing)